tests/check_check_master.c

```diff
@@ -91,194 +93,356 @@ static master_test_t master_tests[] = {
   { "Simple Tests", "test_ck_abort", CK_FAILURE, CK_MSG_TEXT, "Failed" },
   { "Simple Tests", "test_ck_abort_msg", CK_FAILURE, CK_MSG_TEXT, "Failure expected" },
   { "Simple Tests", "test_ck_abort_msg_null", CK_FAILURE, CK_MSG_TEXT, "Failed" },
-  { "Simple Tests", "test_ck_assert", CK_FAILURE, CK_MSG_TEXT, "Assertion 'x == y' failed" },
```

Comment:
I know that these are really long lines, but I do prefer each entry to be on one line. At least for this table, that is. Some of the other long line examples that you have edited are fine.